### PR TITLE
Add dev container to ease development

### PR DESCRIPTION
**Pull request by tetienne**
See https://github.com/custom-components/blueprint

With this dev container, no need anymore to symlink or create a git submodule.

### CODE REVIEW

**Review by vlebourl — APPROVED** [review]
I don't use VSCode, so I trust it's working. Nice addition though

**Comment by tetienne** [comment]
@vlebourl You should have a try, for big project I’m still using Pycharm, but VSCode become a real challenger!

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	"appPort": [
+		"9123:8123"
+	],
+	"postCreateCommand": "container install",
+	"runArgs": [
+		"-v",
+		"${env:HOME}${env:USERPROFILE}/.ssh:/tmp/.ssh"
+	],
+	"extensions": [
+		"ms-python.python",
```
Comment (iMicknl) [inline .devcontainer/devcontainer.json:14]: Could we add PyLance? -> https://devblogs.microsoft.com/python/announcing-pylance-fast-feature-rich-language-support-for-python-in-visual-studio-code/

File: .devcontainer/configuration.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+default_config:
+
+logger:
+  default: error
+  logs:
+    custom_components.blueprint: debug
```
Comment (iMicknl) [inline .devcontainer/configuration.yaml:6]: Change to tahoma.

File: setup.cfg [hunk 3]
```diff
@@ -33,6 +31,5 @@ not_skip = __init__.py
 force_sort_within_sections = true
 sections = FUTURE,STDLIB,INBETWEENS,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 default_section = THIRDPARTY
-known_first_party = homeassistant,tests
-forced_separate = tests
+known_first_party = custom_components.blueprint
```
Comment (iMicknl) [inline setup.cfg:34]: Change to tahoma? (and should we keep homeassistant)

**Review by iMicknl — CHANGES_REQUESTED** [review]
Nice addition! I will try it out soon.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (tetienne) [reply .devcontainer/devcontainer.json:14]: Sure.

File: setup.cfg (hunk 3 above)
Comment (tetienne) [reply setup.cfg]: I’ve removed this. It’s done automatically by isort now.

**Review by iMicknl — APPROVED** [review]

**Comment by tetienne** [comment]
@iMicknl Can you please remove black-format from the expected checks? I have rename it in another PR.

**Comment by iMicknl** [comment]
> @iMicknl Can you please remove black-format from the expected checks? I have rename it in another PR.

Done! ;-)

**Pull request merged**